test/cucumber/StudentConsult.feature

```diff
@@ -0,0 +1,39 @@
+#João Adherval
```

Comment:
Checklist:
Scenarios should reflect the stakeholders needs - OK
Scenario description should strictly correspond to scenario semantics - OK
Look for short scenario steps sentences (use And as needed) - OK
Do not duplicate scenario steps - OK
Do not duplicate scenarios - OK
Scenario should not be vague or incomplete - OK
Specify actions, not non actions or empty actions - OK
Avoid language errors - OK
Use high level actions - OK
Write platform independent scenarios - OK
Focus on testing situation, direct to the point - OK
Focus on testing situation, not general requirement - OK

Cons de todos as features:
- Todo mundo usou nomes diferentes para "nota"
- João e Milena - grades
- Thiago - evaluation
- Danilo - marks